src/Simplex/Chat.hs

```diff
@@ -1724,6 +1733,30 @@ processChatCommand' vr = \case
                 ci <- saveSndChatItem user (CDGroupSnd gInfo) msg (CISndGroupEvent gEvent)
                 toView $ CRNewChatItem user (AChatItem SCTGroup SMDSnd (GroupChat gInfo) ci)
           pure CRMemberRoleUser {user, groupInfo = gInfo, member = m {memberRole = memRole}, fromRole = mRole, toRole = memRole}
+  APIBlockMemberForAll groupId gmId blocked -> withUser $ \user -> do
+    Group gInfo members <- withStore $ \db -> getGroup db vr user groupId
+    let (targetMember, remainingMembers) = partition ((== gmId) . groupMemberId') members
```

Comment:
something like `span` that needs to be used here, not `partition`.

Reply:
`removeQP` in simplexmq is similar, can be done on a normal list too.

Reply:
span/break seem to do a different thing. Could use `find` but then would need to separately filter out element for remaining members